src/test/codegen/swap-small-types.rs

```diff
@@ -12,9 +12,25 @@ type RGB48 = [u16; 3];
 #[no_mangle]
 pub fn swap_rgb48(x: &mut RGB48, y: &mut RGB48) {
     // FIXME MIR inlining messes up LLVM optimizations.
-// WOULD-CHECK-NOT: alloca
-// WOULD-CHECK: load i48
-// WOULD-CHECK: store i48
+    // WOULD-CHECK-NOT: alloca
+    // WOULD-CHECK: load i48
+    // WOULD-CHECK: store i48
```

Comment:
Can those CHECKs be restored or not yet?

Reply:
Not yet, see asm in https://github.com/rust-lang/rust/pull/98824#issuecomment-1173070127

I can implement change suggested in that comment (would fix allocas here) but I want to see opinion about it before diving in.